packages/parse5/lib/parser/index.ts

```diff
@@ -111,7 +111,7 @@ export interface ParserOptions<T extends TreeAdapterTypeMap> {
     onParseError?: ParserErrorHandler | null;
 }
 
-const defaultParserOptions = {
+const defaultParserOptions: Required<ParserOptions<any>> = {
```

Comment:
you could probably type this as the default adapter's map rather than `any`.

when you come to using it in the constructor/bootstrap, though, you'll have to cast it. IMO thats better than this one being untyped though.

whatever types we come up with for tree adapters, this'll always be the case i think. the only way around would be to have some common interface _without_ `treeAdapter` and fallback the adapter separately so it doesn't interfere with the types (you'd still need a cast i think).

Reply:
Made that change 👍